src/components/Editor/Runners/PythonRunner/PythonRunner.jsx

```diff
@@ -14,19 +16,31 @@ const SKULPT_ONLY_MODULES = [
 ];
 
 const PythonRunner = () => {
+  const dispatch = useDispatch();
+
   const project = useSelector((state) => state.editor.project);
+  const activeRunner = useSelector((state) => state.editor.activeRunner);
   const codeRunTriggered = useSelector(
     (state) => state.editor.codeRunTriggered,
   );
   const senseHatAlwaysEnabled = useSelector(
     (state) => state.editor.senseHatAlwaysEnabled,
   );
-  const [usePyodide, setUsePyodide] = useState(true);
+  const [usePyodide, setUsePyodide] = useState(null);
+  const [skupltFallback, setSkulptFallback] = useState(false);
```

Comment:
looks like it's used consistently, but there's a typo here thats makes the code hard to read, `skupltFallback` is probably meant to be `skulptFallback`

Reply:
🤦 🤦 🤦

Reply:
fixed 😄